src/http/structs.rs

```diff
@@ -92,6 +99,64 @@ pub struct EntityDatabasePath {
     pub database: String,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityQueryPath {
+    pub entity: String,
+}
+
+#[derive(Serialize, Deserialize, Clone)]
+pub struct EntityQueryResponse {
+    pub slug: String,
+    pub databases: Vec<EntityDatabase>,
+}
+
+impl EntityQueryResponse {
```

Comment:
I'm not sure whether we should create a trait for _structs that can be converted to tables_, the `generate_table` and `generate_csv` functions are exactly the same as the ones implemented for the query command.

Reply:
Nice idea!

Reply:
One tiny thought: do we care that we lose the slug in the tabular view? We're only keeping the database list, which is probably OK, but figured I'd point it out to see if we care.

Reply:
> do we care that we lose the slug in the tabular view?

Without the database slug what information would be shown to the user?

Reply:
> do we care that we lose the slug in the tabular view?

I've (confusingly) named both the database slugs and the entity slug... slugs in the endpoint. Which one are you referring to?

Reply:
Also, I think I'll create the trait in a separate PR :)
I don't want to clutter this one a lot :p